.changeset/yellow-turkeys-lay.md

```diff
@@ -0,0 +1,11 @@
+---
+"firebase": patch
+"@firebase/firestore-types": patch
+"@firebase/firestore": patch
+"@firebase/webchannel-wrapper": patch
```

Comment:
```suggestion
"@firebase/firestore-types": minor
"@firebase/firestore": minor
"@firebase/webchannel-wrapper": minor
```

We generally no longer include the `firebase` package in this list, since the version bump for the main package will be determined based on the individual version bumps. 

As for whether this is a `patch` or `minor` bump: Any additions to public API surface (or any new feature work that is more than a bug fix) warrant a `minor` release. I think it is safe to say that if the public interface is touched, we always have to bump at least the `minor` version.